src/lib/Theme.ts

```diff
@@ -1680,20 +1679,22 @@ export default class Theme {
     };
 
     private static getImageCdnBaseUrl = async () => {
-        let imageCdnUrl = '';
         try {
-            let startData = {
-                file_name: 'test.jpg',
-                content_type: 'image/jpeg',
-                size: '1',
-            };
-            let startAssetData = (
-                await UploadService.startUpload(
-                    startData,
-                    'application-theme-images',
-                )
-            ).data;
-            return (imageCdnUrl = path.dirname(startAssetData.cdn.url));
+            const dummyFile = path.join(
+                __dirname,
+                '..',
+                '..',
+                'sample-upload'
+            );
+
+            const response = await UploadService.uploadFile(
+                dummyFile,
+                'application-theme-images',
+                null,
+                'image/jpg'
+            );
+
+            return path.dirname(response.complete.cdn.url);
```

Comment:
```suggestion
            return path.dirname(response?.complete?.cdn?.url);
```

Update this `null` check across update places.

Reply:
This is not needed as the response is expected to have this structure. If we do not get assetBasePath, theme sync should actually fail.